[NewProduct] CakePHP

ravage84 opened the pull request.

ravage84 committed:
Add CakePHP product

ravage84 committed:
Correct 3.10 release

usta committed:
Update cakephp.md

ravage84 committed:
Enable active support lcolumn

usta committed:
add license and a bit more info

ravage84 committed:
Remove custom label for EOL column

usta committed:
fix links

ravage84 committed:
Add EOL'd CakePHP 3.x releases

For now in simplified form

ravage84 committed:
Neutral wording

usta commented on products/cakephp.md:
```diff
@@ -0,0 +1,81 @@
+    releaseDate: 2019-12-15
+    latest: "4.4.5"
+    latestReleaseDate: 2022-08-28
+    link: https://bakery.cakephp.org/2019/12/15/cakephp_400_released.html
```
we use links to latest tag's release links

usta replied:
that was the reason i commited this one : https://github.com/endoflife-date/endoflife.date/pull/1672/commits/d596e1d3535aa369372d854716580f9d380ae946

ravage84 replied:
That wasn't clear form the template. Fixed that in https://github.com/endoflife-date/endoflife.date/pull/1674

usta replied:
thanks i merged your PR ( #1674 )

ravage84 replied:
I mistakenly reverted it because of an out of date version of teh file

usta replied:
i know it is completely ok :)

ravage84 replied:
Re-done in https://github.com/endoflife-date/endoflife.date/pull/1672/commits/041dd679b4acbfaaacfb9d95bebf2c942c0375ea

usta commented on products/cakephp.md:
```diff
@@ -0,0 +1,81 @@
+  - releaseCycle: "3.1"
+    eol: true
+    support: false
+  - releaseCycle: "3.0"
```
each releasecycle should have at least releaseDate tag

ravage84 replied:
OK, I'm going to add them

usta replied:
thank you 👍 💯 🥇 🎉 
and if you have time and of course if you could find it , could you also change eol tags to date instead of using true
[sure if you couldnt find them you can leave it as true but we prefer specific dates]

usta replied:
Thank you again ❤️

usta replied:
After this change it is ok to merge so all green after this one :)

ravage84 replied:
> After this change it is ok to merge so all green after this one :)

OK, but I am gonig to add some more stuff, if you don't mind.

usta replied:
Sure :) i was talking about min to be landed more means better :)

usta replied:
btw if cake doesnt have eol dates you can use support dates as its eol dates instead of using true

ravage84 replied:
OK, thanks. But it does. They aren't very clearly documented in all cases, though.

ravage84 committed:
Use latest release links again

https://github.com/endoflife-date/endoflife.date/pull/1672#pullrequestreview-1138011901

ravage84 committed:
Add missing release dates to CakePHP 3.x releases

ravage84 committed:
Add not actively supported CakePHP 4.x releases

In simplified form for now

ravage84 committed:
Add latest CakePHP 4.3.10 release

There is no bakery post, though

ravage84 committed:
The latest CakePHP 4.4 release is actually 4.4.6

But there was not bakery post

usta committed:
add changelogTemplate instead of links to each cycle

ravage84 committed:
Extend PHP support table

usta committed:
add latest tags for each cycle

ravage84 committed:
Add last CakePHP 2.x release

ravage84 committed:
Add last CakePHP 1.3 release

ravage84 committed:
Fix formatting

ravage84 committed:
Set CakePHP 3.9 support date

usta reviewed (approved):
LGTM. Thank you for your contributions 👍 💯 🥇 🎉
